addons/medical_damage/ACE_Medical_Injuries.hpp

```diff
@@ -169,20 +178,23 @@ class ACE_Medical_Injuries {
             class Minor {
                 name = CSTRING(VelocityWound_Minor);
                 minDamage = 0.15;
-                maxDamage = 0.3;
+                maxDamage = 3.5;
                 bleedingRate = 0.025;
+                lethalities[] = {{1, 0.05}, {0, 0.20}};
             };
             class Medium {
                 name = CSTRING(VelocityWound_Medium);
-                minDamage = 0.3;
-                maxDamage = 0.75;
+                minDamage = 3.5;
+                maxDamage = 8;
                 bleedingRate = 0.05;
+                lethalities[] = {{1, 0.15}, {0, 0.60}};
             };
             class Large {
                 name = CSTRING(VelocityWound_Large);
-                minDamage = 0.75;
+                minDamage = 8;
```

Comment:
this seems incredibly high - did you test this value and if so, what were the results?